lib/index.js

```diff
@@ -116,48 +116,6 @@ var nameSpaceIds = {
     CategoryTalk: 15,
 };
 nameSpaceIds.Image = nameSpaceIds.File;
-
-
-/**
- * Checks whether namespace is `Media`
- *
- * @method
- * @name Namespace.prototype#isMedia
- * @return {boolean}
- */
-
-/**
- * Checks whether namespace is `Special`
- *
- * @method
- * @name Namespace.prototype#isSpecial
- * @return {boolean}
- */
-
-/**
- * Checks whether namespace is `Main`
- *
- * @method
- * @name Namespace.prototype#isMain
- * @return {boolean}
- */
-
-/**
- * Checks whether namespace is `Talk`
- *
- * @method
- * @name Namespace.prototype#isTalk
- * @return {boolean}
- */
-
-/**
- * Checks whether namespace is `UserTalk`
- *
- * @method
- * @name Namespace.prototype#isUserTalk
- * @return {boolean}
- */
-
```

Comment:
If you want to retain the comments, make sure to generate them for all 16 or so generated function. So, you prefer me to just get rid of this commit then?